src/target/common/devo/Makefile.inc

```diff
@@ -78,7 +78,7 @@ $(TARGET).dfu: $(TARGET).bin
 ###################################
 $(ODIR)/%.bin: $(ODIR)/%.o_ $(PROTO_LINKFILE) $(PROTO_EXTRA_OBJS) $(TARGET).bin
 	@echo " + Building '$@'"
-	$(LD) -o $(@:.bin=.elf) $< $(PROTO_EXTRA_OBJS) objs/devo7e/uart.o $(LIBOPENCM3) -gc-sections -Map=$(@:.bin=.map) \
+	$(LD) -o $(@:.bin=.elf) $< $(PROTO_EXTRA_OBJS) $(LIBOPENCM3) -gc-sections -Map=$(@:.bin=.map) \
```

Comment:
I suggest first merge #464 as this is also changed in 464 in a more cleaner way.